octopus-agent/src/main/java/octopus/teamcity/agent/OctopusBuildInformationBuildProcess.java

```diff
@@ -115,8 +116,10 @@ else if (OverwriteMode.IgnoreIfExists.name().equals(forcePush)) {
                     commands.add(spaceName);
                 }
 
-                commands.add("--package-id");
-                commands.add(packageId);
+                for(String packageId : StringUtil.split(packageIds, "\n")) {
```

Comment:
Other check for empty lines and strip them out, maybe we want to do that here too?